docs/user_guide/assessment/perform_fairness_assessment.rst

```diff
@@ -276,4 +281,47 @@ overall values for the data:
     count    0.222222
     dtype: float64
 
-In every case, the *largest* difference and *smallest* ratio are returned.
\ No newline at end of file
+In every case, the *largest* difference and *smallest* ratio are returned.
+
+
+.. _assessment_predefined_fairness_metrics:
+
+Predefined fairness metrics
+---------------------------
+
+In addition to the disaggregated analysis of base metrics enabled by
+:class:`MetricFrame`, Fairlearn also provides a set of predefined fairness 
+metrics that output a single score. These metrics take as input 
+`sensitive_features` to compute the maximum difference or ratio between 
+subgroups of a sensitive variable. The predefined fairness metrics offered 
+by Fairlearn are **demographic parity** ratio/difference and **equalized odds** 
+ratio/difference. Note that because these metrics are calculated using 
+aggregations between groups (unlike, say, accuracy, the demographic parity 
```

Comment:
The part in parentheses doesn't quite make sense to me.

Reply:
I removed it for now.